lang/cs/Org.Apache.REEF.Client/YARN/IJobResourceUploader.cs

```diff
@@ -26,7 +26,8 @@ public interface IJobResourceUploader
         /// Upload archived local driver folder to DFS destination path.
         /// </summary>
         /// <param name="driverLocalFolderPath">Local folder where REEF application resources are staged</param>
+        /// <param name="jobSubmissionDirectory">Remote directory path where we will upload resources</param>
         /// <returns>Path, modification time and size of uploaded file as JobResource</returns>
-        JobResource UploadJobResource(string driverLocalFolderPath);
+        JobResource UploadJobResource(string driverLocalFolderPath, string jobSubmissionDirectory);
```

Comment:
Do we need to expose the directory here? The original idea was to keep that information in the configuration of the uploader.

Either way, this is a `public` interface and if possible, we should keep the old signature and maybe `[Obsolete]` it.